### PR TITLE
move to a supported (non depricated) tools container

### PR DESCRIPTION
**Pull request by sferich888**

### CODE REVIEW

File: pkg/oc/cli/debug/debug.go [hunk 1]
```diff
@@ -758,7 +758,7 @@ func (o *DebugOptions) approximatePodTemplateForObject(object runtime.Object) (*
 			}
 		}
 		if len(o.Image) == 0 {
-			image = "registry.access.redhat.com/rhel7/rhel-tools"
+			image = "registry.redhat.io/rhel7/support-tools"
```
Comment (smarterclayton) [inline pkg/oc/cli/debug/debug.go]: This isn't publicly accessible (which was the previous gap).

Comment (sferich888) [reply pkg/oc/cli/debug/debug.go]: Shouldn't a user on 4.x have the ability to pull from registry.redha.io based on the pull-secret they provide at install?

Comment (smarterclayton) [reply pkg/oc/cli/debug/debug.go]: my comment is about non red hat supported users

Comment (sferich888) [reply pkg/oc/cli/debug/debug.go]: Let's allow docs to call out (in our community docs) how to override the image.
@vikram-redhat can you cover this on the community side.

**Pull request merged**